plasmapy/particles/tests/test_parsing.py

```diff
@@ -6,8 +6,7 @@
     InvalidElementError,
     InvalidParticleError,
 )
-from plasmapy.particles.parsing import (
-    # duplicate with utils.pytest_helpers.error_messages.call_string?
+from plasmapy.particles.parsing import (  # duplicate with utils.pytest_helpers.error_messages.call_string?
```

Comment:
This is odd.  I've confirmed `isort` is placing the comment here, which then violates PEP8 max line-length.  I've played with the configuration options, but I couldn't find a solution.  Maybe run `black` after `isort`.

In the end, this comment does not make much sense to me and can probably be deleted.

Reply:
Okay, I'll remove that one. Anything else that needs changing?

Reply:
Nope, everything else looks good to me.